seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -198,14 +198,16 @@ angular.module('BE.seed.controller.mapping', [])
       $scope.isValidCycle = Boolean(_.find(cycles.cycles, {id: $scope.import_file.cycle}));
 
       matching_criteria_columns_payload.PropertyState = _.map(matching_criteria_columns_payload.PropertyState, function (column_name) {
-        var display_name = _.find($scope.mappable_property_columns, {column_name: column_name}).display_name;
+        var display_name = _.find($scope.mappable_property_columns, {column_name: column_name});
+        if (display_name) display_name = display_name.display_name;
         return {
           column_name: column_name,
           display_name: display_name
         };
       });
       matching_criteria_columns_payload.TaxLotState = _.map(matching_criteria_columns_payload.TaxLotState, function (column_name) {
-        var display_name = _.find($scope.mappable_taxlot_columns, {column_name: column_name}).display_name;
+        var display_name = _.find($scope.mappable_taxlot_columns, {column_name: column_name});
+        if (display_name) display_name = display_name.display_name;
```

Comment:
I can't figure out the possible connection between these change and DQ changes. Think you could provide background here?

Reply:
No this is not. This was one of the issues that austin was working on (also seen by Robin during the same time period) that was preventing me from importing a file. This was the work-aound we briefly talked about. It does not directly affect dq check. I will remove it.
![image](https://user-images.githubusercontent.com/49968203/81351903-8503fa00-9082-11ea-8ac9-429851aa3c22.png)